src/charts.js

```diff
@@ -0,0 +1,229 @@
+// import Chart from 'chart.js/auto';
```

Comment:
Is it possible not to merge this file to main yet, as it's not currently in use?